demo/predict-taxi-trip-duration-nb/Dockerfile

```diff
@@ -1,10 +1,137 @@
-FROM centos:8
+# https://github.com/docker-library/openjdk/blob/61bfbb30ae5a0395f08d41107b86bcce90d70ba6/11/jre/slim-bullseye/Dockerfile
+FROM openjdk:11.0.13-jre-slim-bullseye
```

Comment:
I'm open for Debian but is there any reason to switch ?
And code compile on centos ...

Reply:
debian base seems smaller than centos

Reply:
Total image size:
centos:8: 1.63G
openjdk:11.0.13-jre-slim-bullseye: 1.35GB

Reply:
looks like slim did it work